shims/spark301/pom.xml

```diff
@@ -48,7 +48,7 @@
                             <target>
                                 <mkdir dir="${project.build.directory}/extra-resources"/>
                                 <exec executable="bash" failonerror="true" output="${project.build.directory}/extra-resources/spark-${spark301.version}-info.properties">
```

Comment:
In a follow-on we need to generalize this invocation to use spark.version and move to the shim/pom.xml and delete from child shims.